config.yaml

```diff
@@ -0,0 +1,9 @@
+author: Deepak Sharma <deepshar@redhat.com>
```

Comment:
Why should a config has author?

Reply:
https://github.com/spf13/cobra/blob/master/cobra/README.md#configuring-the-cobra-generator

Reply:
Removed non required config fields